### PR TITLE
Complex Radio

### PR DESCRIPTION
**Pull request by TylerCousins**
Fixed Everything

### CODE REVIEW

File: Complex Radio/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "author": {
+      "name": "Tyler",
+      "id": "351991098387988491"
+    },
+    "service": "Complex_Radio",
```
Comment (Alanexei) [inline Complex Radio/dist/metadata.json:6]: Remove the underscore from the service name.

**Comment by TylerCousins** [comment]
> You've made no changes.

I downloaded the latest typescript and compiled.

**Comment by Bas950** [comment]
> > You've made no changes.
> 
> I downloaded the latest typescript and compiled.

that doesnt mean you changed the settings, download the fork locally and it should give you the correct settings for the workspace

**Comment by TylerCousins** [comment]
> > > You've made no changes.
> > 
> > 
> > I downloaded the latest typescript and compiled.
> 
> that doesnt mean you changed the settings, download the fork locally and it should give you the correct settings for the workspace

What fork

**Comment by Bas950** [comment]
> > > > You've made no changes.
> > > 
> > > 
> > > I downloaded the latest typescript and compiled.
> > 
> > 
> > that doesnt mean you changed the settings, download the fork locally and it should give you the correct settings for the workspace
> 
> What fork

bruh https://github.com/Tyler-Cousins/Presences

**Comment by TylerCousins** [comment]
> > > You've made no changes.
> > 
> > 
> > I downloaded the latest typescript and compiled.
> 
> that doesnt mean you changed the settings, download the fork locally and it should give you the correct settings for the workspace

Is that ok now?

**Pull request merged**